study.md

```diff
@@ -21,18 +21,22 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+The ordering of JS just confuses me. I am understanding it at a better rate but still confusing.  I don't comprehend the design of the problems but I think I found a tutor. Example:
+
+var myFunction = function [name]([param1[, param2[, ..., paramN]]]) {
```

Comment:
Brackets usually indicate something that's optional.  Here, it means that name, param1, param2, param3, etc. are all optional when declaring a function with a function expression.  Thus, ```var myFunction = function () { ... }``` is just as valid as ```var myFunction = function someFunctionName(someParameter, anotherParameter) { ... }```.